.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow uses actions that are not certified by GitHub. They are provided
+# by a third-party and are governed by separate terms of service, privacy
+# policy, and support documentation.
+
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  workflow_dispatch:
+  schedule:
+    - cron: '36 1 * * 6'
```

Comment:
nit: Add a comment with the schedule deciphered, like: Every Saturday at 1:36 UTC

Reply:
there's probably more stuff that's not needed, but I guess this workflow was generated, and we may leave it that way..

Reply:
yes, I agree. I can go both ways, though.

Reply:
Done.